src/commands/misc.rs

```diff
@@ -14,40 +14,44 @@ struct Misc;
 
 #[command]
 #[description("Teste de conectividade entre o Bot e os servidores do Discord.")]
-pub fn ping(ctx: &mut Context, msg: &Message) -> CommandResult {
+pub async fn ping(ctx: &Context, msg: &Message) -> CommandResult {
     msg.channel_id
-        .say(&ctx.http, "Pong but in <:rust:530449316607688724>!")?;
+        .say(&ctx.http, "Pong but in <:rust:530449316607688724>!")
+        .await?;
     Ok(())
 }
 
 #[command]
 #[description(
     "Informação relativa à linguagem de programação utilizada para desenvolvimento do Bot."
 )]
-pub fn info(ctx: &mut Context, msg: &Message) -> CommandResult {
-    msg.channel_id.say(&ctx.http, "Powered by JAVA8™")?;
+pub async fn info(ctx: &Context, msg: &Message) -> CommandResult {
+    msg.channel_id.say(&ctx.http, "Powered by Rust™").await?;
```

Comment:
change to an equivalent shitpost
the level of shitpost must remain constant between version